client-go/informers/externalversions/jobset/v1alpha2/jobset.go

```diff
@@ -15,24 +15,24 @@ limitations under the License.
 package v1alpha2
 
 import (
-	"context"
+	context "context"
```

Comment:
can you please revert all those files?

Reply:
If we disagree with this I think we gotta fix this upstream. Otherwise everything someone needs to revert this whenever they do generate.